selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -63,7 +63,7 @@ def update(self, sm):
     v_ego = sm['carState'].vEgo
 
     v_cruise_kph = sm['controlsState'].vCruise
-    v_cruise_kph = min(v_cruise_kph, V_CRUISE_MAX)
+    v_cruise_kph = max(min(v_cruise_kph, V_CRUISE_MAX), V_CRUISE_MIN)
```

Comment:
I wonder if we're always giving a small amount of acceleration on non-pcm cars on first emable (now would be brake)